content/en/docs/tasks/debug-application-cluster/debug-application.md

```diff
@@ -58,7 +58,7 @@ Again, the information from `kubectl describe ...` should be informative.  The m
 
 * Make sure that you have the name of the image correct.
 * Have you pushed the image to the registry?
-* Run a manual `docker pull <image>` on your machine to see if the image can be pulled.
+* Try to manually pull the image on your machine to see if the image can be pulled.
```

Comment:
"on your machine" now reads strange, maybe drop it?

```suggestion
* Try to manually pull the image to see if the image can be pulled.
```

Reply:
I'd prefer to add context. The `docker` tool remains widely used.

```suggestion
* Try to manually pull the image to see if the image can be pulled. For example,
  if you use Docker on your PC, run `docker pull <image>`. Does that work OK?
```